daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -137,10 +137,9 @@ General guidance on upgrading can be found for [self hosted mode]({{< ref self-h
 | 1.9.0                    |                   N/A |                    1.9.6 |
 | 1.10.0                   |                   N/A |                   1.10.8 |
 | 1.11.0                   |                   N/A |                   1.11.4 |
-| 1.12.0                   |                   N/A |                   1.12.4 |
-| 1.12.0 to 1.13.0         |                   N/A |                   1.13.4 |
-| 1.13.0                   |                   N/A |                   1.13.4 |
-| 1.13.0 to 1.14.0         |                   N/A |                   1.14.0 |
+| 1.12.0 to 1.12.4         |                   N/A |                   1.13.5 |
+| 1.13.0 to 1.13.5         |                   N/A |                   1.14.0 |
+| 1.14.0                   |                   N/A |                   1.14.0 |
```

Comment:
Remove this. Redundant
```suggestion
```

Reply:
Can you also update 1.9, 1.10 and 1.11 to have the same "to"
So 
1.9.0 to 1.9.6   -> 1.10.8
1.10.0 to 1.10.8 -> 1.11.4
1.11.0 to 1.11.4 -> 1.12.4